appsync/aws-appsync-events/src/main/java/com/amplifyframework/aws/appsync/events/Events.kt

```diff
@@ -37,12 +37,13 @@ class Events @VisibleForTesting internal constructor(
     val endpoint: String,
     val connectAuthorizer: AppSyncAuthorizer,
     val defaultChannelAuthorizers: ChannelAuthorizers,
-    options: Options,
-    okHttpClient: OkHttpClient
+    okHttpClient: OkHttpClient,
+    loggerProvider: LoggerProvider?,
```

Comment:
Why remove `Options` here instead of making it `private val`? Seems it would be simpler to keep it as the number of options potentially grows.

Reply:
Pushed change. @mattcreaser can you reapprove